go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -464,6 +464,28 @@ func newSharedGatewayOpenFlowManager(nodeName, macAddress, gwBridge, gwIntf stri
 	}
 	nFlows++
 
+	if config.Gateway.DisableSNATMultipleGWs {
```

Comment:
Why is this check here in shared gateway but not in local gateway?

Reply:
In local gw mode we are always disable snat for multiple gateways. RH is the only one that uses local gw mode, and we always want it disabled.